Userland/Libraries/LibJS/Runtime/RegExpConstructor.cpp

```diff
@@ -93,4 +131,69 @@ JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::symbol_species_getter)
     return vm.this_value(global_object);
 }
 
+JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::input_setter)
+{
+    auto new_value = vm.argument(0);
+    global_object.regexp_constructor()->m_input = new_value;
+    return new_value;
+}
+
+// RegExp.input / RegExp.$_ [[Get]]
+JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::input_getter)
+{
+    return global_object.regexp_constructor()->m_input;
+}
+
+// RegExp.lastMatch / RegExp.$& [[Get]]
+JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::last_match_getter)
+{
+    return global_object.regexp_constructor()->m_last_match;
+}
+
+// RegExp.lastParen / RegExp.$+ [[Get]]
+JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::last_paren_getter)
+{
+    return global_object.regexp_constructor()->m_last_paren;
+}
+
+// RegExp.leftContext / RegExp.$` [[Get]]
+JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::left_context_getter)
+{
+    return global_object.regexp_constructor()->m_left_context;
+}
+
+// RegExp.rightContext / RegExp.$' [[Get]]
+JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::right_context_getter)
+{
+    return global_object.regexp_constructor()->m_right_context;
+}
+
+ThrowCompletionOr<Value> RegExpConstructor::get_captured_value(VM& vm, [[maybe_unused]] GlobalObject& global_object, size_t index)
```

Comment:
get_foo() is usually not compliant with Serenity's code style, instead, it should be just foo()